### PR TITLE
Added feature to disallow shipping to PO Box

### PR DESCRIPTION
**Pull request by johnmccombs1**

### CODE REVIEW

File: app/code/community/Bolt/Boltpay/controllers/ShippingController.php [hunk 1]
```diff
@@ -383,4 +388,28 @@ public function getPrefetchCacheIdentifier($quote, $addressData)
...
+    {
+        return Mage::getStoreConfig('payment/boltpay/allow_po_box');
+    }
+
+    /**
+     * @param $address1
+     * @param $address2
+     * @return mixed
+     */
+    protected function doesAddressContainPOBox($address1, $address2)
```
Comment (daisy1754) [inline app/code/community/Bolt/Boltpay/controllers/ShippingController.php]: @johnmccombs1 This is a good candidate for testing. Can you write a unit test?

Comment (daisy1754) [reply app/code/community/Bolt/Boltpay/controllers/ShippingController.php]: same input proposal (please add more)

match
 - P.O. Box 66
 - Post Box 123
 - Post Office Box 456

not match
 - Post street
 - 2 Box

Comment (johnmccombs1) [reply app/code/community/Bolt/Boltpay/controllers/ShippingController.php]: These checks have been added along with a few more.

**Comment by johnmccombs1** [comment]
@daisy1754 can you review the latest unit test changes.

**Pull request merged**